sovrin_client/cli/command.py

```diff
@@ -67,13 +75,13 @@
     examples="send proofreq Over-21 to JaneDo")
 
 showFileCmd = Command(
-    id="show file",
+    id="show",
```

Comment:
Is "show" really enough of an ID? It seems like we have several "show" commands...